### ISSUE TITLE
weird pdf produced by motif_with_stats

### ISSUE THREAD

**Issue by dorothyzh** [nanoporetech/tombo#106]
Hi when I try to test the motif_with_stats,
"tombo plot motif_with_stats --fast5-basedirs fast5 --statistics-filename sample_de_novo_detection.tombo.stats --genome-fasta model.fa --motif NNNNN"

The pdf it produced is weird, could you help this problem? 

Thanks 

[tombo_results.motif_statistics.pdf](https://github.com/nanoporetech/tombo/files/2326392/tombo_results.motif_statistics.pdf)

**Comment by marcus1487** [comment]
I had noticed this in development at some point. I think that upgrading my `ggplot2` installation resolved this issue.

**Comment by marcus1487** [comment]
TL;DR For those coming across this post, in order to solve this weird plot bug follow these steps to obtain a correct plot: 1) install `cowplot`, 2) clone Tombo from github, 3) change [this value](https://github.com/nanoporetech/tombo/blob/master/tombo/R_scripts/plotMotifStats.R#L4) to `TRUE` and 4) run `motif_with_stats` again to fix this plotting issue.

I have just come across this issue again and I've not been able to find a robust solution using ggplot2 recommended fixes (e.g. here: https://cran.r-project.org/web/packages/egg/vignettes/Ecosystem.html). I have added a flag in the motif_with_stats R script [here](https://github.com/nanoporetech/tombo/blob/master/tombo/R_scripts/plotMotifStats.R#L4) that can be set in order to use cowplot (but I'm still not adding it as a dependency).
